t68/variant1.ts

```diff
@@ -0,0 +1,55 @@
+document.querySelectorAll('.T68').forEach((button) => {
+  show(button);
+});
+
+document.addEventListener('cover.ready T68', (event) => {
+  show(event.target);
+});
+
+function show(button) {
+  console.debug('fix styling', button);
+
+  button.classList.remove('u-hidden');
+  button.parentElement.querySelector('.AddToCart').classList.add('u-hidden');
+
+  button.style.minWidth = '120px';
```

Comment:
you should make a generic function to add or change css properties of a specific element to avoid unnecesary code, something like:

`export interface IIndexable {
    [key: string]: any;
}`

`function updateHtmlElement(element: HTMLElement, property, value: string | number | whatever you want) {
     (element as IIndexable)[property] = value;
}`

Reply:
- separate content from design. In your case separate content from design using a specific function to handle the styling
- easy maintanable; maybe you want to add something custom or some nested properties

And the list could go on